### ISSUE TITLE
Delete command bug

### ISSUE THREAD

**Issue by Snorrrlax** [MystK/delta-bot-three#187]
Every time we send a delete command to DeltaBot for this comment, it removes it but adds it again a few moments later:

https://www.reddit.com/r/changemyview/comments/627uu2/cmv_people_are_too_attached_to_their_personal/dfkumm2/

**Comment by MystK** [comment]
similar to #140 but unrelated.

This happens because of the "rescans." A fix for #140 would fix this.

Suggested solution: Make DeltaBot add a comment.

**Comment by hacksoncode** [comment]
Should this still be open? Or has it been overcome by another bug having to do with using a database (which only recently became possible due to the server upgrade).

I'm actually more in favor of MystK's suggestion of having DB leave a comment in cases where we remove a delta... to explain that it was removed intentionally, and maybe leave an appeal link...

**Comment by MystK** [comment]
Yes still an issue I believe. We need to modify DB3 to leave a comment/notesaying that this delta was manually removed(hidden or public doesn't matter)
________________________________
